hack/local-up-master/lib.sh

```diff
@@ -243,6 +243,17 @@ function localup::start_kubecontrollermanager() {
     os::log::debug "Waiting for kube-controller-manager to come up"
     kube::util::wait_for_url "http://localhost:10252/healthz" "kube-controller-manager: " 1 ${WAIT_FOR_URL_API_SERVER} ${MAX_TIME_FOR_URL_API_SERVER} \
         || { os::log::error "check kube-controller-manager logs: ${KUBE_CONTROLLER_MANAGER_LOG}" ; exit 1 ; }
+
+    # we need SCCs as they are part of the OpenShift apiserver bootstrap process
+    echo "Waiting for the SCCs to appear"
```

Comment:
don't we have shell helpers for this stuff?

Reply:
I was not able to successfully source the library containing it